Snakefile

```diff
@@ -24,7 +24,14 @@ rule prepare_example:
     shell:
         "cp -r examples/{wildcards.scenario}/preprocessed results/{wildcards.scenario}/preprocessed"
 
-
+rule prepare_conv_pp:
+    input:
+        opsd = "raw/conventional_power_plants_DE.csv", gpkg = "raw/boundaries_germany_nuts3.gpkg"
```

Comment:
```suggestion
        opsd = "raw/conventional_power_plants_DE.csv",
        gpkg = "raw/boundaries_germany_nuts3.gpkg"
```

Reply:
On my computer it only works if both inputs are on the same line, seperated by a comma.
If I put gpkg on a new line as you suggested, I get this Error:
```
SyntaxError in line 30 of C:\Users\meinm\Documents\Git\oemof-B3\Snakefile:
Unexpected keyword gpkg in rule definition (Snakefile, line 30)
```

Reply:
Indent has to be 4 spaces